internal/osbuild/systemd_journald_stage.go

```diff
@@ -0,0 +1,104 @@
+/**
+Include the new journald config stage to configure journald to
+persist the journal.Instead of creating the
+directory `/var/log/journal`, we explicitly configure journald
+via the new stage.
+This is also what Fedora CoreOS like:
+https://github.com/coreos/fedora-coreos-config/blob/d709e4f5fc3970fb6b938dc7bcfdb8d602d2db94/overlay.d/05core/usr/lib/systemd/journald.conf.d/10-coreos-persistent.conf
```

Comment:
This comment is phrased like a commit message.  We generally don't need this kind of comment in the stages here, but if we're including one, it makes more sense to have the same information as the osbuild counterpart.

Reply:
I think, it can be deleted too and added in osbuild instead.